### PR TITLE
chore: 0.11.12 post-release

### PR DESCRIPTION
**Pull request by fraidev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,42 +2,44 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## Platform Version 0.11.13-dev - UNRELEASED
+
 ## Platform Version 0.11.12-dev - 2024-09-13
```
Comment (digikata) [inline CHANGELOG.md:7]: We missed in in the 0.11.12 release, it  should be 0.11.12 with no "-dev"

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-0.11.12
\ No newline at end of file
+0.11.12-dev-1
```
Comment (digikata) [inline VERSION:1]: shouldnt this be 0.11.13-dev-1?

**Review by digikata — APPROVED** [review]
Couple of minor nits to fix

File: VERSION (hunk 2 above)
Comment (fraidev) [reply VERSION]: Nice catch! Fixed, thanks!!!

File: CHANGELOG.md (hunk 1 above)
Comment (fraidev) [reply CHANGELOG.md]: Also fixed!

**Pull request merged**